cpp/src/examples/gcylon/shuffle_perf.cpp

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
lets not have an example called perf, it would be suffle and tere can be an option to measure perf

Reply:
i renamed shuffle_perf to gshuffle and conditionally output the result to the file